File: TestArgent/TestArgent/StandMenager/Detecteurdepoussiere.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TestArgent
{
    class Detecteurdepoussiere : Usine
    {
        public Detecteurdepoussiere()
        {
            niveau = 0;
            this.gainArgent =100;
            this.tempsDeProduction = 30;
            this.nom = "Detecteur de Poussiere";
            this.coutAmelioration = 1000;
            acheter = false;
            prerequis = "Balayage rapide NIV 15";
            numUsine = 2;
        }
    }
}

File: Projet1/Tycoon/Detecteurdepoussiere.cpp
#include "Detecteurdepoussiere.h"

Detecteurdepoussiere::Detecteurdepoussiere() {
    niveau = 0;
    gainArgent = 100;
    tempsDeProduction = 30;
    nom = "Detecteur de Poussiere";
    coutAmelioration = 1000;
    acheter = false;
    prerequis = "Balayage rapide NIV 15";
    numUsine = 2;
}